src/components/shared/Entity/WrapperWithHeader/Header.tsx

```diff
@@ -0,0 +1,27 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { AccordionSummary, Fade, Typography } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import { BaseComponentProps } from 'types';
+
+interface Props extends BaseComponentProps {
+    expanded: boolean;
+}
+
+function Header({ children, expanded }: Props) {
```

Comment:
Moved to a stand alone component to prepare for setting a `ref` for a `slide` transition.